packages/eslint-plugin-material-ui/src/rules/no-hardcoded-labels.js

```diff
@@ -15,13 +15,19 @@ module.exports = {
       Literal(node) {
         const canLabelComponent =
           node.parent.type === 'JSXElement' ||
+          node.parent.type === 'JSXExpressionContainer' ||
```

Comment:
Could you add a test for this change so that we understand why this was changed?

Reply:
I would rather revert the change to be honest

Reply:
Sure.